File: Beginner/Age.c
#include<stdio.h>
void main()
{
 int n;
 printf("Enter a number: ");
 scanf("%d",&n);
 if(n>18)
 {
  printf("Adult");
 }
 else if(n>10 && n<=18) 
 { 
  printf("Teen");
 }
 else
 { 
  printf("child");
 }
 return 0;
 }

File: Beginner/age.py
while True:
    try:
        a = int(input("enter your age :"))
        if a > 18:
            print("Adult")
        elif 10 < a <= 18:
            print("Teen")
        elif a <= 10:
            print("Child")
        break
    except ValueError:
        print("enter valid age")
        break
